src/Features/Workflows/WorkflowCard/WorkflowCard.tsx

```diff
@@ -107,22 +107,21 @@ const WorkflowCard: React.FC<WorkflowCardProps> = ({ teamId, quotas, workflow })
 
   const menuOptions = [
     {
-      itemText: "Edit Workflow",
+      itemText: "Edit",
```

Comment:
Rearranged these and updated the copy per a request. I think this makes more sense